public/locales/en/translation.json

```diff
@@ -64,13 +100,49 @@
         "id": 0
       },
       {
-        "title": "cooking training",
+        "title": "Cooking class",
         "name": "Dameon Peterson",
         "description": "Certe, inquam pertinax non quo quaerimus, non fuisse torquem detraxit hosti et quidem exercitus quid ex eo ortum, tam crudelis fuisse, nihil molestiae consequatur, vel illum, qui officia deserunt mollitia animi, id omnia referri oporteat.",
         "address": "Istanbul, Turkey",
         "phoneNumber": "+906467291373",
         "date": "27/10/2020",
         "id": 1
+      },
+      {
+        "title": "Child psychology course",
+        "name": "Dameon Peterson",
+        "description": "Certe, inquam pertinax non quo quaerimus, non fuisse torquem detraxit hosti et quidem exercitus quid ex eo ortum, tam crudelis fuisse, nihil molestiae consequatur, vel illum, qui officia deserunt mollitia animi, id omnia referri oporteat.",
+        "address": "Istanbul, Turkey",
+        "phoneNumber": "+906467291373",
+        "date": "27/10/2020",
+        "id": 2
+      },
+      {
+        "title": "Graphic design for beginners",
+        "name": "Dameon Peterson",
+        "description": "Certe, inquam pertinax non quo quaerimus, non fuisse torquem detraxit hosti et quidem exercitus quid ex eo ortum, tam crudelis fuisse, nihil molestiae consequatur, vel illum, qui officia deserunt mollitia animi, id omnia referri oporteat.",
+        "address": "Istanbul, Turkey",
+        "phoneNumber": "+906467291373",
+        "date": "27/10/2020",
+        "id": 3
+      },
+      {
+        "title": "UI/UX training course",
+        "name": "Dameon Peterson",
+        "description": "Certe, inquam pertinax non quo quaerimus, non fuisse torquem detraxit hosti et quidem exercitus quid ex eo ortum, tam crudelis fuisse, nihil molestiae consequatur, vel illum, qui officia deserunt mollitia animi, id omnia referri oporteat.",
+        "address": "Istanbul, Turkey",
+        "phoneNumber": "+906467291373",
+        "date": "27/10/2020",
+        "id": 4
+      },
+      {
+        "title": "Montessori education",
```

Comment:
it would be much better if it was a Montessori methods course or something similar since it's a method of education, [source ](https://en.wikipedia.org/wiki/Montessori_education)

Reply:
+1

Reply:
done